modules/sfm/README.md

```diff
@@ -0,0 +1,52 @@
+Structure From Motion module
+============================
+
+This module contains algorithms to perform 3d reconstruction from 2d images. The core of the module is a light version of [libmv](http://code.google.com/p/libmv/) which is a Structure from Motion (SfM) library divided into different modules (image/detector/descriptor/multiview) that allow to resolve part of the SfM process.
+
+The main reconstruction API currently uses the open-source [Ceres Solver](http://ceres-solver.org/) in order to solve part of the Bundle Adjustment plus the points Intersect. If Ceres Solver is not installed on your system, the reconstruction funcionality will be disabled.
+
+
+Ceres installation (Linux)
+--------------------------
+
+Here are the instructions on how to install the Ceres Solver on your machine (Linux).
+
+    sudo apt-get install libceres-dev
```

Comment:
I didn't do. This is my output of libceres-dev, maybe you can get some clues:

eriba@gari-linux:~/software/opencv/build$ apt-cache policy libceres-dev 
libceres-dev:
  Instaŀlat: 1.8.0-1+trusty2
  Candidat:  1.8.0-1+trusty2
  Taula de versió:
 *** 1.8.0-1+trusty2 0
        500 http://packages.ros.org/ros/ubuntu/ trusty/main amd64 Packages
        100 /var/lib/dpkg/status